tests/Globe.test.js

```diff
@@ -1,12 +1,7 @@
-import { Globe } from "../src/og/Globe";
 import { Worker } from "./worker";
```

Comment:
Is it ok? looks  empty?)

Reply:
look at fool [file](https://github.com/openglobus/openglobus/blob/pipline_optimization/tests/Globe.test.js)  - it's unused import